Add beta 8 release notes and features overview (#22)
withstudiocms/studiocms.dev: src/content/blog/beta-8-release.md

---
title: "Beta 8 Release"
description: "StudioCMS 0.1.0-beta.8 is being released."

> blueSky:
>   postId: 3lfyk3gw6kk26

Reviewer: ```suggestion
```
This can be left out for now until we make the post, Adam will add it back in

    image: "./assets/beat-8-release.png"
    alt: "Beta 8 Release"
author: paul-valladares
---

> Hey peeps, as we roll into 2025, we're super excited to drop StudioCMS version `0.1.0-beta.8`. This release builds on all the hard work we did in 2024 and brings a ton of cool improvements to help both devs and marketers easily handle content in the Astro ecosystem. Let’s break down what’s new.

Reviewer 2: Sentence terminates
```suggestion
Hey peeps! As we roll into 2025, we're super excited to drop StudioCMS version `0.1.0-beta.8`. This release builds on all the hard work we did in 2024 and brings a ton of cool improvements to help both devs and marketers easily handle content in the Astro ecosystem. Let’s break down what’s new.
```

> As we step into 2025, StudioCMS is excited to announce the release of version `0.1.0-beta.8`, a major milestone that builds on our progress from 2024. This update introduces significant enhancements, making it easier for developers to manage content within the Astro ecosystem. Let’s explore what’s new and how it can benefit you.

Reviewer: ```suggestion
As we step into 2025, StudioCMS is excited to announce the release of version `0.1.0-beta.8`, a major milestone that builds on our progress from 2024. This update introduces significant enhancements, making it easier for developers and marketers alike to manage content within the Astro ecosystem. Let’s explore what’s new and how it can benefit you.
```

## What’s New in Beta.8

StudioCMS `0.1.0-beta.8` brings several exciting features:

> **Redesigned Dashboard:** The dashboard has been completely revamped using the `@studiocms/ui` library, offering a modern, intuitive interface for managing content.

Reviewer 2: Should there be a link to ui?

Author: i don't think so? what do you guys think? @Adammatthiesen @louisescher

Reviewer 3: wouldn't hurt IMO

> **Redesigned Dashboard:** The dashboard has been completely revamped using the `@studiocms/ui` library, offering a modern, intuitive interface for managing content.

Reviewer 3: ```suggestion
**Redesigned Dashboard:** The dashboard has been completely revamped using the [`@studiocms/ui`](https://ui.studiocms.dev) library, offering a modern, intuitive interface for managing content.
```

> **Redesigned Dashboard:** The dashboard has been completely revamped using the `@studiocms/ui` library, offering a modern, intuitive interface for managing content. This unexpected improvement enhances usability, making navigation smoother for developers.

Reviewer: This isn't really unexpected now, is it
```suggestion
**Redesigned Dashboard:** The dashboard has been completely revamped using the `@studiocms/ui` library, offering a modern, intuitive interface for managing content.
```

> **Auth System Overhaul:** Security is tighter with a new authentication system using lucia-next, new encryption (requiring a CMS_ENCRYPTION_KEY), and support for multiple oAuth providers. Login and signup pages have a fresh look with backgrounds, improving user experience.

Reviewer: More a comment than anything, and a question directed @Adammatthiesen - I don't think this is new?

Reviewer 3: Auth system is 100% rebuilt from the ground... including the need for an encryption key

**Headless CMS Model:** StudioCMS now focuses on backend functionalities, giving you flexibility to integrate with any frontend framework. For headful setups, the `@studiocms/blog` plugin is recommended, offering a complete solution.

**Dynamic Sitemaps:** This feature automatically generates sitemaps, improving your website’s search engine optimization (SEO) and ensuring content is discoverable.

> 

Reviewer: Also not a new feature I don't think? CC @Adammatthiesen

Reviewer 3: this is new and part of the plugin system

**User Management Enhancements:** New features for creating users and sending invites simplify team collaboration, with an updated UI for better clarity.

> **Plugin System and i18n:** A basic plugin system allows custom extensions, and internationalization (i18n) support makes StudioCMS accessible to a global audience, with tools like a LanguageSelector component.

Reviewer: ```suggestion
**Plugin System and i18n:** A basic plugin system allows custom extensions, and internationalization (i18n) support makes StudioCMS accessible to a global audience.
```

These updates reflect our commitment to providing a robust, flexible CMS for developers.

## Comparative Analysis: Beta.7 vs. Beta.8

To highlight the advancements, here’s a comparison of key features between the last release, `0.1.0-beta.7`, and the new `0.1.0-beta.8`, based on available release notes and changelog:

| Feature | Beta.7 (Sep 24, 2024) | Beta.8 (Feb 27, 2025) |
| --- | --- | --- |

> | Authentication System | Lucia-based, with basic oAuth support | Overhauled with lucia-next, new encryption, multiple oAuth |

Reviewer: ```suggestion
| Authentication System | Lucia-based, with basic oAuth support | Overhauled with lucia-next |
```
Based on previous comments

| Dashboard | Standard UI, basic navigation | Completely redesigned using `@studiocms/ui`, modern interface |
| CMS Model | Headful, integrated routes | Headless, with `@studiocms/blog` for headful setups |
| Sitemaps | Static integration | Dynamic sitemaps for improved SEO |
| User Management | Basic user creation | Enhanced with invites, updated UI |
| Plugin System | Limited, basic support | Basic system introduced, expandable |

> | Internationalization | Minimal, no dedicated system | Full i18n support with LanguageSelector |

Reviewer: ```suggestion
| Internationalization | Minimal, no dedicated system | Full i18n support with |
```

> This table, derived from the GitHub releases and pull request, illustrates the significant upgrades in beta.8, enhancing functionality and user experience.

Reviewer: ```suggestion
This table, derived from the GitHub releases and pull requests, illustrates the significant upgrades in beta.8, enhancing functionality and user experience.
```

## What’s Next & How to Get Involved

> Looking ahead, we’re all about making StudioCMS even more robust. Expect more focus on stability, performance, and a killer user experience. We’re planning to expand our plugin ecosystem, improve documentation, and keep the community vibe strong—just check out our Discord if you haven’t already. A big shoutout to [turso](https://turso.tech) for their sponsorship and to everyone who’s been part of this journey.

Reviewer: Last little typo fix, other than that LGTM
```suggestion
Looking ahead, we’re all about making StudioCMS even more robust. Expect more focus on stability, performance, and a killer user experience. We’re planning to expand our plugin ecosystem, improve documentation, and keep the community vibe strong—just check out our Discord if you haven’t already. A big shoutout to [Turso](https://turso.tech) for their sponsorship and to everyone who’s been part of this journey.
```

> Looking ahead, we’re all about making StudioCMS even more robust. Expect more focus on stability, performance, and a killer user experience. We’re planning to expand our plugin ecosystem, improve documentation, and keep the community vibe strong—just check out our Discord if you haven’t already. A big shoutout to [Turso](https://turso.tech) for their sponsorship and to everyone who’s been part of this journey.

Reviewer 2: Should probably use the specific studiocms link here

Reviewer: We have a referral link iirc? Should be the one on the website or from our docs

Reviewer 3: @dreyfus92 I think this is what he means

`and keep the community vibe strong—just check out our [Discord](https://chat.studiocms.dev) if you haven’t already.`

Reviewer 3: Oh and we should be using the `https://tur.so/studiocms` link instead of their main link!

Reviewer 2: Yeah, the Turso link

> Looking ahead, we’re all about making StudioCMS even more robust. Expect more focus on stability, performance, and a killer user experience. We’re planning to expand our plugin ecosystem, improve documentation, and keep the community vibe strong—just check out our Discord if you haven’t already. A big shoutout to [Turso](https://turso.tech) for their sponsorship and to everyone who’s been part of this journey.

Reviewer 3: ```suggestion
Looking ahead, we’re all about making StudioCMS even more robust. Expect more focus on stability, performance, and a killer user experience. We’re planning to expand our plugin ecosystem, improve documentation, and keep the community vibe strong—just check out our [Discord](https://chat.studiocms.dev) if you haven’t already. A big shoutout to [Turso](https://tur.so/studiocms) for their sponsorship and to everyone who’s been part of this journey.
```
